Common/GPU/Vulkan/VulkanQueueRunner.h

```diff
@@ -54,38 +54,38 @@ enum class PipelineFlags : u8 {
 ENUM_CLASS_BITOPS(PipelineFlags);
 
 // Pipelines need to be created for the right type of render pass.
-enum RenderPassType {
+// TODO: Rename to RenderPassType? It is currently a bitfield.
```

Comment:
s/Type/Flag[s]/?

-[Unknown]

Reply:
Yeah oops, this evolved - I started by renaming then decided not to, for now.